/*

   calin/proto/iact_data/functional_event_visitor.proto -- Stephen Fegan -- 2016-04-14

   Protobufs for configuration of functional event visitors

   Copyright 2016, Stephen Fegan <sfegan@llr.in2p3.fr>
   LLR, Ecole polytechnique, CNRS/IN2P3, Universite Paris-Saclay

   This file is part of "calin"

   "calin" is free software: you can redistribute it and/or modify it
   under the terms of the GNU General Public License version 2 or
   later, as published by the Free Software Foundation.

   "calin" is distributed in the hope that it will be useful, but
   WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
   General Public License for more details.

*/

syntax = "proto3";

import "calin.proto";

package calin.ix.iact_data.functional_event_visitor;

message FixedWindowSumFunctionalTelescopeEventVisitorConfig {
  int32 integration_0                                      = 1 [
    (CFO).desc = "Start of integration window in samples." ];
  uint32 integration_n                                     = 2 [
    (CFO).desc = "Size of integration window in samples." ];
};
